probe-rs/src/debug/mod.rs

```diff
@@ -3,6 +3,10 @@
 //! The `debug` module contains various debug functionality, which can be
 //! used to implement a debugger based on `probe-rs`.
 
+// Bad things happen to the VSCode debug extenison and debug_adapter if we panic at the wrong time.
+#![deny(clippy::unwrap_used, clippy::panic, clippy::expect_used)]
```

Comment:
Same thing applies to those as in the comments in the `debug.rs`.

Reply:
Done